public/team.html

```diff
@@ -612,6 +612,23 @@ <h3>Mohit Madhav Rao</h3>
                         </div>
                     </div>
                
+                    <div class="col-md-6 col-lg-3 mb-4">
+                        <div class="team-member">
+                            <figure>
+                                <ul class="social">
+                                    <li><a href="#"><span class="icon-github"></span></a></li>
```

Comment:
Is this line needed?

Reply:
she is not having github account, incase she makes one will add in future so didn't delete the line😁

Reply:
But won't that show a blank icon presently?